app/forms/concerns/work_form_behavior.rb

```diff
@@ -68,6 +68,16 @@ def model_attributes(params)
       #  -- they don't have these fields in the form at all so they don't get repopulated.
       clean_params = encode_physical_container(params, clean_params)
       clean_params = encode_external_id(params, clean_params)
+
+      clean_params.keys.each do |key|
```

Comment:
I wonder if we want to do this earlier; below the call to super?

Also it might be nice for this to be its own method; this one is getting a little long.